transport_nantes/topicblog/test_tn_links.py

```diff
@@ -51,10 +51,14 @@ def test_buttons(self):
         self.assertEqual(self.parser.transform('[[don:fixed|5]]((give!))'), \
                          don.fixed_amount_donation_button(5, 'give!'))
 
-    def test_news(self):
-        settings.csrf_token = 'some text that must exist'
-        self.assertEqual(self.parser.transform('[[news:kangaroo]]((aardvark))'), \
-                         render_inclusion_tag_to_html('newsletter', 'show_mailing_list'))
+    ## Disabled due to captcha matching.
```

Comment:
Faced the captcha validation problem with the login form, turns out there is a way to get away with it in tests. 
 Can see here how it's done : 
 https://github.com/transport-nantes/tn_web/commit/e077ef0173a990cbfc76d25367972e682043913b#diff-e4c08575b99cb05e4808251bfb547bc1a429237cd6af2ac65d313cf1e59c02b0

TLDR; Captcha has a [test mode](https://django-simple-captcha.readthedocs.io/en/latest/advanced.html#captcha-test-mode)

Reply:
Yeah, I looked at that.  But it makes the captcha accept whatever it gets so that the submit validates, it doesn't make it produce the same string.  And here I was testing that the two functions generate the same html.

Reply:
Ha, got it. Indeed, seems like we shall have to do without it.